src/vue/backend/src/views/Home.vue

```diff
@@ -1,18 +1,17 @@
 <template>
```

Comment:
This is test code that should not be committed to the repo. If could cause unnecessary merge conflicts as we build out new components